src/pages/home/components/header/index.js

```diff
@@ -3,7 +3,7 @@ import { CheckCircleIcon } from '@chakra-ui/icons';
 import { Box, Button, Heading, HStack, Text } from '@chakra-ui/react';
 import { BsBarChart } from 'react-icons/bs';
 import { MdOutlineSettings } from 'react-icons/md';
-import { FaUserCircle } from 'react-icons/fa';
+import { FaUserCircle, FaHome, FaTasks } from 'react-icons/fa';
```

Comment:
Suggestion:  since react icons have different imports based on the suffix type so you can add a index file in assets under icon and import all icons that you are using and then import them from that one file wherever you required them. It will make your icons import to one line.